utils_linux.go

```diff
@@ -240,7 +241,15 @@ func (r *runner) run(config *specs.Process) (int, error) {
 			return -1, err
 		}
 	}
-	if r.detach {
+	if !r.create {
+		if err := process.Signal(syscall.SIGCONT); err != nil {
+			r.terminate(process)
+			r.destroy()
+			tty.Close()
+			return -1, err
+		}
+	}
+	if r.detach || r.create {
```

Comment:
Are you asking a question or what are you trying to get at? 

No, the create process does not have to stick around to reap, the proc calling create can handle that and orchestrate the entire thing from create to destroy.